index.js

```diff
@@ -2,12 +2,52 @@ const mongoose = require('mongoose');
 const Recipe = require('./models/Recipe.model'); // Import of the model Recipe from './models/Recipe.model.js'
 const data = require('./data.js'); // Import of the data from './data.js'
 
+// mongoose.set('useNewUrlParser', true);
+// mongoose.set('useFindAndModify', false);
+// mongoose.set('useCreateIndex', true);
+// mongoose.set('useUnifiedTopology', true);
+
 // Connection to the database "recipeApp"
 mongoose
   .connect('mongodb://localhost/recipe-app-dev', {
     useCreateIndex: true,
     useNewUrlParser: true,
     useUnifiedTopology: true
   })
-  .then(x => console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`))
+  .then(x =>
+    console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`)
+  )
   .catch(err => console.error('Error connecting to mongo', err));
+
+// // Recipe.create({
```

Comment:
instead of commenting to avoid overlapping your insertions you could have tried before the create method clean the collection everytime your run your code.
Something like this:
```
Recipe.collection.drop()
```
This way you will clean your collection at the beginning of everytime your run the code

Reply:
yes! I learned it, thank you for feedback :)